app/api/events_role_permission.py

```diff
@@ -0,0 +1,40 @@
+from flask_rest_jsonapi import ResourceDetail, ResourceList, \
+    ResourceRelationship
+
+from app.api.bootstrap import api
+from app.api.schema.event_role_permissions import EventsRolePermissionSchema
+from app.models import db
+from app.models.permission import Permission
+
+
+class EventsRolePermissionList(ResourceList):
+    """
+    List Events Role Permission
+    """
+    decorators = (api.has_permission('is_admin', methods="GET"),)
+    methods = ['GET']
+    schema = EventsRolePermissionSchema
+    data_layer = {'session': db.session,
+                  'model': Permission}
+
+
+class EventsRolePermissionDetail(ResourceDetail):
+    """
+    Events Role Permission detail by id
+    """
+    schema = EventsRolePermissionSchema
+    decorators = (api.has_permission('is_admin', methods=["GET", "PATCH"]),)
+    methods = ['GET']
```

Comment:
Please check it's super field. Here is the comment:
```
#: A list of methods this view can handle.
    methods = None
```

Is PATCH request allowed on this endpoint. If yes, then also add it in it. If it is not allowed then remove it from here:
`decorators = (api.has_permission('is_admin', methods=["GET", "PATCH"]),)`